test/EFCore.PG.FunctionalTests/Scaffolding/NpgsqlDatabaseModelFactoryTest.cs

```diff
@@ -1374,12 +1378,14 @@ public void Warn_missing_principal_table_for_foreign_key()
                     Assert.Empty(dbModel.Sequences);
 
                     // Now make sure the field itself is properly reverse-engineered.
-                    var column = dbModel.Tables.Single().Columns.Single();
-                    Assert.Equal(ValueGenerated.OnAdd, column.ValueGenerated);
-                    Assert.Null(column.DefaultValueSql);
-                    Assert.Equal(NpgsqlValueGenerationStrategy.SerialColumn, (NpgsqlValueGenerationStrategy)column[NpgsqlAnnotationNames.ValueGenerationStrategy]);
+                    foreach (var column in dbModel.Tables.Select(t => t.Columns.Single()))
+                    {
+                        Assert.Equal(ValueGenerated.OnAdd, column.ValueGenerated);
+                        Assert.Null(column.DefaultValueSql);
+                        Assert.Equal(NpgsqlValueGenerationStrategy.SerialColumn, (NpgsqlValueGenerationStrategy)column[NpgsqlAnnotationNames.ValueGenerationStrategy]);
+                    }
                 },
-                @"DROP TABLE ""SerialSequence""");
+                @"DROP TABLE ""SerialSequence""; DROP SCHEMA my_schema CASCADE");
```

Comment:
Do you mean to drop the new `serial_sequence` table here too?

Reply:
Good catch, thanks!